README.md

```diff
@@ -76,6 +76,8 @@ GDS is an advocate of responsible vulnerability disclosure. If you’ve found a
 
 For full details on how to tell us about vulnerabilities, [see our security policy](https://github.com/alphagov/govuk-frontend/security/policy).
 
+The govuk-frontend repository is public. We do not monitor it outside of office hours (10am to 4pm). To learn how to report inappropriate content, see [GitHub's page on reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). If you're not sure whether some content is appropriate, check it against our [code of conduct](https://github.com/alphagov/govuk-frontend/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
I don't think this belongs under the 'Security' heading. It potentially needs its own section where we can point to the CoC first?

I also wonder if we might want to try and differentiate between two separate things here:

- Activity on our repos which violates our CoC, but that we would expect users to report to our team to be resolved within our working hours – for example, a discussion that's got 'a little out of hand' / heated
- Activity on our repos which clearly violates GitHub's baseline CoC / terms of service, for example spam or pornography, which need a quicker response and should be reported to GitHub so that they can deal with it 'immediately', even outside of our working hours.

At the minute, we talk about getting in touch with GitHub but then tie it back to our own CoC:

> If you're not sure whether some content is appropriate, check it against our [code of conduct](https://github.com/alphagov/govuk-frontend/blob/main/CODE_OF_CONDUCT.md).

Maybe we could update the current 'Contribution guidelines' section at the bottom? What about something like this?

> ## Contributing
> 
> This repository is public and we welcome contributions from anyone.
> 
> Contributors to repositories within the alphagov organisation are expected to follow the [Contributor Covenant Code of Conduct](https://github.com/alphagov/.github/blob/main/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct), and those working within Government are also expected to follow the [Civil Service Code](https://www.gov.uk/government/publications/civil-service-code/the-civil-service-code).
> 
> We are not able to monitor activity on this repo outside of office hours (10am to 4pm UK time). You can [report abuse or spam to GitHub](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) for a faster response outside of these hours.

Reply:
@36degrees Currently, the [end of the README](https://github.com/alphagov/govuk-frontend/blob/f07569c9e7724606970c0d2764f1e9ce7fa14092/README.md#contribution-guidelines) is:

> ## Contribution guidelines
> If you want to help us build GOV.UK Frontend, view our [contribution guidelines](https://github.com/alphagov/govuk-frontend/blob/f07569c9e7724606970c0d2764f1e9ce7fa14092/CONTRIBUTING.md).

The "contribution guidelines" link is to a doc titled 'Contributing'. So, if we're retitling this section 'Contributing', should we change the title of the linked doc to 'Contribution guidelines'?

Suggested text looks good. Would just tweak:

> ## Contributing
> 
> [To learn how to help us build GOV.UK Frontend, see our contribution guidelines.](https://github.com/alphagov/govuk-frontend/blob/f07569c9e7724606970c0d2764f1e9ce7fa14092/CONTRIBUTING.md)
> 
> The govuk-frontend repository is public and we welcome contributions from anyone.
> 
> Contributors to alphagov repositories are expected to follow the [Contributor Covenant Code of Conduct](https://github.com/alphagov/.github/blob/main/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct). Contributors working within government are also expected to follow the [Civil Service code](https://www.gov.uk/government/publications/civil-service-code/the-civil-service-code).
> 
> We're unable to monitor activity on this repository outside of our office hours (10am to 4pm, UK time). To get a faster response at other times, you can [report abuse or spam to GitHub](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).

Reply:
Looks good to me!

Reply:
@36degrees Thanks! Have added the content to this PR, and similar content to PRs for the [govuk-design-system](https://github.com/alphagov/govuk-design-system/pull/1975) and [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit/pull/1142) READMEs.